src/rp2_common/hardware_adc/include/hardware/adc.h

```diff
@@ -88,6 +88,16 @@ static inline void adc_select_input(uint input) {
     hw_write_masked(&adc_hw->cs, input << ADC_CS_AINSEL_LSB, ADC_CS_AINSEL_BITS);
 }
 
+/*! \brief  ADC get input
```

Comment:
"ADC get input" is probably a bit ambiguous? It could either mean "get the input channel" or "get the input value" ? :wink: